Dockerfile

```diff
@@ -120,6 +120,9 @@ RUN patch -d / -p0 < /tmp/HTCONDOR-503.add-sl-support.patch
 COPY hosted-ce/overrides/HTCONDOR-451.allow-batch_gahp.patch /tmp
 RUN [[ $BASE_YUM_REPO != 'release' ]] || patch -d / -p0 < /tmp/HTCONDOR-451.allow-batch_gahp.patch
 
+COPY hosted-ce/ssh-to-login-node /usr/local/bin
+RUN chmod +x /usr/local/bin/ssh-to-login-node
```

Comment:
I thought `COPY` handled proper copying of permissions during the build

Reply:
Good point.